### PR TITLE
Setup: apply BC velocity compensation

### PR DESCRIPTION
**Pull request by kulakovri**
I've applied compensation on southern, eastern and western boundaries with initially uneven velocity parameters set up.

After 200 steps mass balance was not violated:

image
image

### CODE REVIEW

File: MDLIB/mdoodz-private.h [hunk 1]
```diff
@@ -561,4 +561,12 @@ void            SetBCs(SetBCs_ff setBCs, MdoodzInput *instance, grid *mesh);
 
 void            ValidateSetup(MdoodzSetup *setup, MdoodzInput *instance);
 
+typedef struct {
+  double VxWestSum;
+  double VxEastSum;
+  double VzSouthSum;
```
Comment (halterw) [inline MDLIB/mdoodz-private.h:567]: what about VzNorthSum? - to keep MDOODZ universally applicable? :)

**Review by halterw — APPROVED** [review]

File: MDLIB/mdoodz-private.h (hunk 1 above)
Comment (kulakovri) [reply MDLIB/mdoodz-private.h:567]: I don't need VzNorthSum here. It's a struct that we will use to pass in several parameters in one function argument

**Comment by kulakovri** [comment]
image

**Comment by kulakovri** [comment]
found the bug, wip

**Comment by kulakovri** [comment]
image

**Comment by kulakovri** [comment]
image

**Comment by kulakovri** [comment]
image

**Comment by kulakovri** [comment]
image

**Comment by kulakovri** [comment]
image

**Comment by kulakovri** [comment]
I revisit this, but no point in it being a Pull Request

**Pull request closed without merging**